wcomponents-theme/src/test/spec/wc.ui.redirect.test.mjs

```diff
@@ -0,0 +1,81 @@
+import redirect from "wc/ui/redirect.mjs";
+
+describe("wc/ui/redirect", () => {
+	// window = Object.create(Window);
+
+	const delay = 50;  // milliseconds to wait for events and stuff to be actioned
+	const origWindow = window.parent;
+	const replaceSpy = jasmine.createSpy();
+
+	beforeEach(() => {
+		window.parent = {
+			location : {
+				storedUrl : "",
```

Comment:
Regarding `storedUrl` fair solution. I would personally not do it this way mainly because `window.location` actually has a [defined interface](https://html.spec.whatwg.org/multipage/nav-history-apis.html#the-location-interface) and the property `storedUrl` is not part of it right?

Functionally no problem but some of the tools will get angry.
I would simply move `storedUrl` off the object and into the `beforeEach` function as a variable (with `let` not `const`).